public/index.html

```diff
@@ -0,0 +1,557 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Not a Number</title>
+  <meta charset="utf-8">
+  <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>💻</text></svg>">
+<style>
```

Comment:
inline svg favicon for minimal resources impact, we will have to use NaN logo here at some point.